charts/replicated-library/templates/lib/_names.tpl

```diff
@@ -71,8 +71,8 @@ If release name contains chart name it will be used as a full name.
 {{/* Get name of current app */}}
 {{- define "replicated-library.names.appname" -}}
   {{- $name := include "replicated-library.names.name" . -}}
-  {{- if hasKey . "AppName" -}}
-    {{- $name = .AppName -}}
+  {{- if hasKey . "ObjectName" -}}
+    {{- $name = .ObjectName -}}
```

Comment:
I changed this from AppName to ObjectName. I couldn't actually find an AppName built in object maybe I'm missing something?

The result of this on my dev chart is that this function always just returns the ChartName. This resulted in label selectors using the ChartName which would cause collisions in selection if multiple apps were in the same chart.

Changing this to ObjectName retuns the name of the app. Now the label selectors use the application name and the othe objects use the chart name. I think that was the original intention but I'm not sure what AppName is from so I might have missed something with it.

Reply:
Yeah, i think this was just something that I missed on the recently restructuring i did for the multiple release in same namespaces work.